dev_docs/tutorials/performance/running_performance_journey_in_cloud.mdx

```diff
@@ -0,0 +1,93 @@
+Cloud deployments.
+The process takes a few steps:
+- Create a cloud deployment
+- Re-configure deployment with APM enabled and reporting metrics to the monitoring cluster
+- Create a user with `superuser` role to run tests with
+- Checkout the branch that matches your cloud deployment version
+- Run the performance journey
+
+### Re-configure deployment for Kibana and Elasticsearch
+We use [kibana-ops-e2e-perf](https://kibana-ops-e2e-perf.kb.us-central1.gcp.cloud.es.io/) cluster to monitor performance testing.
+
+If you would like to report APM metrics to this cluster, copy `SECRET_TOKEN` and `SERVER_URL` values from [packages/kbn-journeys/journey/journey_apm_config.ts](https://github.com/elastic/kibana/blob/60c82765779419d356a131e212682b69b035804b/packages/kbn-journeys/journey/journey_apm_config.ts#L10-L11)
+
+#### Change Elasticsearch configuration
+In the ESS Admin Console, find your deployment and navigate to `Security` page. Click `Add Settings` under `Elasticsearch keystore` and add new entry:
+
+```
+Setting name: tracing.apm.secret_token
+Secret: <SECRET_TOKEN>
+```
+
+Navigate to `Advanced Edit` page and change `Deployment Configuration` by adding the following JSON object to `resources.elasticsearch.plan.elasticsearch`:
+
+```
+"user_settings_override_json": {
+  "tracing.apm.enabled": "true",
+  "tracing.apm.environment": "development",
+  "tracing.apm.agent.service_name": "elasticsearch",
+  "tracing.apm.agent.server_url": "<SERVER_URL>",
+  "tracing.apm.agent.metrics_interval": "120s",
+  "tracing.apm.agent.transaction_sample_rate": "1"
+}
+```
+
+Save changes and make sure cluster is restarted successfully.
+
+#### Change Kibana configuration
+Navigate to `Advanced Edit` page and change `Deployment Configuration` by adding the following JSON object to `resources.kibana.plan.kibana`:
+
+```
+"user_settings_override_json": {
+  "elastic.apm.active": true,
+  "elastic.apm.breakdownMetrics": false,
+  "elastic.apm.captureBody": "all",
+  "elastic.apm.captureRequestHeaders": "true",
+  "elastic.apm.captureSpanStackTraces": false,
+  "elastic.apm.centralConfig": false,
+  "elastic.apm.contextPropagationOnly": "false",
+  "elastic.apm.environment": "development",
+  "elastic.apm.globalLabels.deploymentId": "<DEPLOYMENT_ID>",
+  "elastic.apm.globalLabels.journeyName": "<YOUR_JOURNEY_NAME>",
+  "elastic.apm.longFieldMaxLength": "300000",
+  "elastic.apm.metricsInterval": "120s",
+  "elastic.apm.propagateTracestate": true,
+  "elastic.apm.sanitizeFieldNames": "password,passwd,pwd,secret,*key,*token*,*session*,*credit*,*card*,*auth*,set-cookie,pw,pass,connect.sid",
+  "elastic.apm.secretToken": "<SECRET_TOKEN>",
+  "elastic.apm.serverUrl": "<SERVER_URL>",
+  "elastic.apm.transactionSampleRate": 1
+}
+```
+
+Note: DEPLOYMENT_ID and YOUR_JOURNEY_NAME values are optional labels to find the APM traces for your run.
+
+Save changes and make sure cluster is restarted successfully.
```

Comment:
Can the above kibana and elastic settings be changed together? In such way, the cluster is only needed to be restarted once.

Reply:
Yes, it is. Originally I thought it is easier to re-configure in steps to check if deployment still healthy. But since it is expected to work, I agree with doing it with 1 restart.